FirebaseMLModelDownloader/Sources/TelemetryLogger.swift

```diff
@@ -0,0 +1,163 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import FirebaseCore
+import GoogleDataTransport
+
+/// Extension to set Firebase app info.
+extension SystemInfo {
+  mutating func setAppInfo(apiKey: String?, projectID: String?) {
+    appID = Bundle.main.bundleIdentifier ?? "unknownBundleID"
+    // appVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String ?? "unknownAppVersion"
+    appVersion = "1"
```

Comment:
I wonder if Firebase SDK version is more important than the app version which don't control and cannot make much sense on most of the cases? 

Firebase version is available via `FirebaseVersion()` func.

Reply:
On Android, this is the version of the developer's app so I'm logging that at this point for consistency. I'll add a TODO here to revisit logging this.

Reply:
I'm curious how it is used. I guess, it may be used for A/B testing? If so, then makes sense. The only nit - a constant [CFBundleShortVersionString](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleshortversionstring) should be used instead of the string literal.

Reply:
Thanks I'll look into this some more. For now I've added a TODO.